packages/flutter/lib/src/material/stepper.dart

```diff
@@ -563,6 +566,12 @@ class _StepperState extends State<Stepper> with TickerProviderStateMixin {
     );
   }
 
+  List<Widget> _buildVerticalControlsBody(int index) {
+    return widget.useControl
+        ? <Widget>[widget.steps[index].content, _buildVerticalControls()]
+        : <Widget>[Row(children: <Widget>[Container(child: widget.steps[index].content)])];
```

Comment:
the row and the container seem like they should be doing nothing here, what is their purpose?

Reply:
The row and container was added to keep the Stepper layout looking fine, without it the Stepper breaks it's alignment as showed in the bellow image:
The same issue happens if we make 'controlsBuilder' return `SizedBox.shrink`

![screen shot 2018-12-01 at 14 47 30](https://user-images.githubusercontent.com/4993276/49330655-a650e480-f578-11e8-8f82-f758b10e967c.png)



![screenshot_1543682947](https://user-images.githubusercontent.com/4993276/49330627-478b6b00-f578-11e8-910c-6bf447be02f6.png)